tests/data/test_pandas_helpers.py

```diff
@@ -180,3 +180,36 @@ def test_string_with_time_stripped_first(self, base):
         input = base + "_Value"
         expected = base + "_Time"
         assert expected == undertest.event_time(input)
+
+
+class TestEventName:
+    @pytest.mark.parametrize("suffix", ["", "_Time", "_Value"])
+    @pytest.mark.parametrize(
+        "base",
+        [
+            ("A"),
+            ("1"),
+            ("normal_column"),
+            ("4reallyl0ngc*lum\namewithsymbo|s"),
+        ],
+    )
+    def test_three_suffixes_align(self, base, suffix):
+        input = base + suffix
+        assert base == undertest.event_name(input)
+
+    @pytest.mark.parametrize(
+        "input, expected",
+        [
+            (None, None),
```

Comment:
Can we add the obvious default test at the top (the bottom two tests are exceptional, even though they also handled the default case)

Reply:
Done, if I understood correctly